src/encauth/ccm/ccm_memory.c

```diff
@@ -380,7 +380,9 @@ int ccm_memory(int cipher,
    fastMask = 0;
 #endif
    mask = 0;
-   zeromem(skey,   sizeof(*skey));
+   if (skey != uskey) {
+      zeromem(skey,   sizeof(*skey));
```

Comment:
There is already `if (skey != uskey)` couple of lines above (line 334).

Should not it be better to move `zeromem(skey,   sizeof(*skey))` there?

Reply:
so you mean like this?
```
    cipher_descriptor[cipher].done(skey);
#ifdef LTC_CLEAN_STACK
    zeromem(skey,   sizeof(*skey));
#endif
```
would be fine for me

Reply:
Yes, that's it.